### DISCUSSION TITLE
Feature Request - Set the NDI source to transparent when feed is black

### DISCUSSION THREAD

**Discussion by ShawnDeviron** [DistroAV/DistroAV#1110] [Ideas]
I am using DistroAV in a two computer setup for streaming via OBS. If I close my OBS on my main computer (or my computer crashes, or loses network connectivity for whatever reason), it would be nice for the NDI source to return to the transparent background that it was prior to making the connection with the main computer. This way I can put an image behind it in OBS and have my stream view that image.

**Comment by Trouffman** [comment]
This should be implemented soon via : https://github.com/DistroAV/DistroAV/pull/1276
